tools/gfx/cuda/render-cuda.cpp

```diff
@@ -1176,7 +1176,11 @@ class CUDADevice : public RendererBase
 public:
     virtual SLANG_NO_THROW SlangResult SLANG_MCALL initialize(const Desc& desc) override
     {
-        SLANG_RETURN_ON_FAIL(slangContext.initialize(desc.slang, SLANG_PTX, "sm_5_1"));
+        SLANG_RETURN_ON_FAIL(slangContext.initialize(
+            desc.slang,
+            SLANG_PTX,
+            "sm_5_1",
```

Comment:
Is sm_5_1 an suitable profile for CUDA? If it doesn't need a profile can I just pass nothing?